settings.py

```diff
@@ -968,6 +968,8 @@
 DATADOG_API_KEY = None
 DATADOG_APP_KEY = None
 
+TEST_SHOULD_USE_SQL_BACKEND = False
+
```

Comment:
Can you elaborate a bit on this setting?  When do you expect it to be enabled or disabled?